File: backend/Models/TouristPlace.cs
namespace OrtegaTourism.Models
{
    public class TouristPlace
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Category { get; set; } = string.Empty;
        public string ImageUrl { get; set; } = string.Empty;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public List<string> Activities { get; set; } = new List<string>();
        public string OpeningHours { get; set; } = string.Empty;
    }
}

File: backend-php/Models/TouristPlace.php
<?php

class TouristPlace {
    public int $id = 0;
    public string $name = '';
    public string $description = '';
    public string $category = '';
    public string $imageUrl = '';
    public float $latitude = 0;
    public float $longitude = 0;
    public array $activities = [];
    public string $openingHours = '';
    
    public function toArray(): array {
        return [
            'id' => $this->id,
            'name' => $this->name,
            'description' => $this->description,
            'category' => $this->category,
            'imageUrl' => $this->imageUrl,
            'latitude' => $this->latitude,
            'longitude' => $this->longitude,
            'activities' => $this->activities,
            'openingHours' => $this->openingHours
        ];
    }
}
